### PR TITLE
add manual client regression testing script

### PR DESCRIPTION
**Pull request by syberry-aotyutsky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,3 +134,12 @@ make format
...
 ```
+
+#### Running test scripts
+
+Manual testing scripts could be found at `tests/regression`.
+In order to run one, i.e. `client.py`, follow the steps:
+
+1. copy corresponding `.env.dist` file as `.env` file (if you plan to run `client.py`, copy `client.env.dist` as `client.env`)
+2. fill variables within `.env` file
+3. launch the script (while being in the `tests/regression` folder, execute `python client.py`)
```
Comment (stu-speed) [inline README.md:145]: (crux-python) ➜  regression git:(testing) ✗ pwd
/Users/stuartlai/crux-python/tests/regression
(crux-python) ➜  regression git:(testing) ✗ python client.py
Traceback (most recent call last):
  File "/Users/stuartlai/crux-python/tests/regression/client.py", line 4, in <module>
    from crux import Crux, TRACE
ModuleNotFoundError: No module named 'crux'
Needs export PYTHONPATH

File: tests/regression/client.env.dist [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# please copy the file as 'client.env' and fill the variables, only CRUX_API_KEY is mandatory
+CRUX_API_KEY=
+CRUX_DATASET_ID=
+CRUX_RESOURCE_ID=
```
Comment (stu-speed) [inline tests/regression/client.env.dist:4]: Here is code snipit so resource_id is not needed
    dataset = cx.get_dataset(dataset_id)
    log.info("Dataset: %s", dataset.name)
    fs = dataset.get_latest_files()
    for f in fs:
        print(f.supplier_implied_dt, f.id, f.frame_id, f.name, f.ingestion_time)

Comment (syberry-aotyutsky) [reply tests/regression/client.env.dist:4]: resourceid isn't a mandatory parameter for client.env - if it's not set, script searches for available id in Python client. same for CRUX_DATASET_ID

File: README.md (hunk 1 above)
Comment (syberry-aotyutsky) [reply README.md:145]: on my setup i didn't need to export PYTHONPATH to run the script, so could you please tell me how should i update the readme block ?

Comment (stu-speed) [reply README.md:145]: No thats ok.  I ran direct rather than install crux.

**Pull request merged**